src/uncrustify.cpp

```diff
@@ -479,6 +480,36 @@ int main(int argc, char *argv[])
       }
    }
 
+   /* Set config options using command line arguments.*/
+   idx = 0;
+   while ((p_arg = arg.Params("--set", idx)) != NULL)
+   {
+      char buffer[256];
+      strcpy(buffer, p_arg);
```

Comment:
I'd rather we used the right function instead of tried to maintain an old style.